ansible/configs/sap-smart/sat_workload.yml

```diff
@@ -0,0 +1,126 @@
+- name: 'Install required packages'
+  yum:
+    name: '{{ item }}'
+    state: installed
+  loop:
+    - satellite
+    - firewalld
+    - python-firewall
+
+- name: 'Ensure firewalld is running'
+  service:
+    name: firewalld
+    state: started
+    enabled: yes
+
+- name: 'Open Firewall for Satellite use'
+  firewalld:
+    service: "{{ item }}"
+    permanent: yes
+    state: enabled
+    immediate: yes
+  loop:
+    - http
+    - https 
+    - ssh
+
+- name: "Install Satellite with default parameters"
+  command: > 
+    satellite-installer
+      --scenario satellite
+      --foreman-initial-organization "{{ satellite_organization }}"
+      --foreman-initial-location "{{ satellite_location }}"
+      --foreman-initial-admin-username "{{ satellite_username }}"
+      --foreman-initial-admin-password "{{ satellite_password }}"
+      --certs-cname "sat-{{ guid }}.{{ guid }}.{{ osp_cluster_dns_zone }}"
+      --foreman-proxy-dns-managed=false
+      --foreman-proxy-dhcp-managed=false
+
+# - name: "Setup Satellite as disconnected mode"
+#   command: >
+#     hammer settings set --name content_disconnected --value true
+
+- name: Prevent Satellite to connect to access.redhat.com
+  lineinfile:
+    path: /etc/hosts
+    line: 127.0.0.1 access.redhat.com
+    create: yes
+
+- name: "Copy subscription manifest .zip file"
+  copy:
+    src: "{{ manifest_file_path }}"
+    dest: /root/manifest.zip
+
+- name: "Add manifest to Satellite"
+  command: >
```

Comment:
Can you look at your use of command module like this? You are using folding block operator, but in some you are leaving the `\` between lines and others you aren't. If you aren't using literal block operator, I don't think you need the `\`.

Reply:
Yes, just removed that, not sure why I added the operator in the first place when is not required